go/vm/test/processor_test.go

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package vm_test
+
+// This file contains a few initial shake-down tests or a Processor implementation.
+// Right now, the tested features are minimal. Follow-up work is needed to systematically
+// establish a set of test cases for Processor features.
+//
+// TODO:
+// - test gas price charging
+// - test gas refunding
+// - test left-over gas refunding
+// - test recursive calls
+// - test roll-back on revert
+// - improve test setup
```

Comment:
I guess this comment would cover converting these to TDT.

Reply:
yes